Source/Core/CorridorGeometry.js

```diff
@@ -597,7 +597,7 @@ define([
             length = topNormals.length;
 
             var extrudeNormals = new Float32Array(length * 6);
-            for (i = 0; i < length; i ++) {
+            for (var i = 0; i < length; i ++) {
```

Comment:
If `i` is also used below, either move `var i` above to its own statement or rename it to `j`, etc. below.